### DISCUSSION TITLE
How can we run CefSharp with different version of CEF redist?

### DISCUSSION THREAD

**Discussion by n303ef** [cefsharp/CefSharp#4750] [Q&A] [answered]
Currently CefSharp can't work with different version of CEF redist. If I try to put it, application crashes. But if I put any CEF redist within same major version, it works fine.
I would like to run some early CEF builds (for now, version 123), for which there is no according CefSharp version.
I've looked into commits when CefSharp major version is updated (for example, from 120 to 121), and found only version number changes. I tried to do it myself, with its source code, I've successfully built it, but surpisingly it still crashes with CEF version that I expect to work, and still works with previous version. Yes, there can be some breaking changes, but I also tried versions that already confirmed to not having them (for testing), and still failed in my attempts.
What am I doing wrong?

**Comment by amaitland** [comment, marked as answer]
You need to **upgrade** to a new version of CEF (a new build of `libcef_dll_wrapper` is required).

1. Build packages using https://github.com/cefsharp/cef-binary (using build.ps1)
2. Upgrade to new version using https://github.com/cefsharp/CefSharp/blob/master/UpdateNugetPackages.ps1

Execute the powershell scripts with the version number that you are targeting.
